### ISSUE TITLE
Make the "Run" button in Epoch functional

### ISSUE THREAD

**Issue by cmnrd** [lf-lang/lingua-franca#1083]
Currently, hitting the "Run" button in Epoch only produces the message "The selection cannot be launched, and there are no recent launches." It would be great if we could instead actually run the produced binary from Epoch.

**Comment by lhstrh** [comment]
I agree with @cmnrd. @a-sr, would this be difficult to do? From what I recall, creating a run configuration is a manual effort. Could it be done programmatically?

**Comment by a-sr** [comment]
Yes, it should be possible, somehow, but I have no experience with that.
I found two tutorials (https://www.vogella.com/tutorials/EclipseLauncherFramework/article.html and http://codeandme.blogspot.com/2013/11/debugger-2-launch-framework-not-only.html) and it looks like a lot of work, especially getting all the Eclipse specific registration right and supporting the launch configuration UI.
Furthermore, if we want to do it right, we have to consider our different target languages. For example, in Python we do not produce an executable; federated programs neither.
We would also have to ensure that the started binary is up to date, which probably requires weaving in the compilation, resulting in an alternative to the current compile button.
